src/syscheckd/db/src/db.cpp

```diff
@@ -7,31 +7,50 @@
  */
 
 #include "dbsync.hpp"
+#include "dbsync.h"
 #include "db.h"
 #include "fimCommonDefs.h"
 #include "fimDB.hpp"
 #include "fimDBHelper.hpp"
 #include <thread>
+#include "dbFileItem.hpp"
+
+#ifdef WIN32
+#include "dbRegistryKey.hpp"
+#include "dbRegistryValue.hpp"
+#endif
+
+struct CJsonDeleter
+{
+    void operator()(char* json)
+    {
+        cJSON_free(json);
+    }
+    void operator()(cJSON* json)
+    {
+        cJSON_Delete(json);
+    }
+};
+
 
 #ifdef __cplusplus
 extern "C" {
 #endif
 
-
 /**
  * @brief Create the statement string to create the dbsync schema.
  *
  * @return std::string Contains the dbsync's schema for FIM db.
  */
-const char* CreateStatement()
+static char *CreateStatement()
```

Comment:
Why do you use a C declaration with a C++ function? You could use only C++ here!

Reply:
It will be fixed in this issue https://github.com/wazuh/wazuh/pull/11383